parsely/parselyandroid/ParselyTracker.java

```diff
@@ -258,18 +307,27 @@ public void trackPlay(String url, @NonNull ParselyVideoMetadata videoMetadata) {
         this.videoEngagementManager.start();
     }
 
+    /*! \brief Pause video tracking.
+     *
+     * Stops engagement tracking an ongoing video. If `trackPlay` is immediately called again for
+     * the same video, a new video start event will not be sent. This models a user pausing a
+     * playing video.
+     */
     public void trackPause() {
         if (this.videoEngagementManager == null) {
-            PLog("No ongoing video to stop.");
             return;
         }
-        PLog("trackPause called");
         this.videoEngagementManager.stop();
     }
 
+    /*! \brief Reset tracking on a video.
+     *
+     * Stops engaged time tracking and resets state for video tracking. This means that if
```

Comment:
Unnecessary distinction here between "engaged time tracking" and "video tracking"
> Stops video tracking and resets internal state.